packages/main/src/i18n/messagebundle.properties

```diff
@@ -43,6 +43,12 @@ CAROUSEL_PREVIOUS_ARROW_TEXT=Previous Page
 # Carousel Next Page text
 CAROUSEL_NEXT_ARROW_TEXT=Next Page
 
+#XFLD: Label of the container holding the color swatches
```

Comment:
swatches -> just "the colors" ;) The people who translate these will not know what this word means